Data/Map/Strict/Internal.hs

```diff
@@ -1379,7 +1379,7 @@ mapWithKey f (Bin sx kx x l r) =
 #endif
 
 -- | /O(n)/.
--- @'traverseWithKey' f m == 'fromList' <$> 'traverse' (\(k, v) -> (\v' -> v' `seq` (k,v')) <$> f k v) ('toList' m)@
+-- @'traverseWithKey' f m == 'fromList' <$> 'traverse' (\(k, v) -> (\v' -> v' ``seq`` (k,v')) <$> f k v) ('toList' m)@
```

Comment:
I don't think we want links for Prelude basics like `seq`. The only exceptions (maybe) are for list functions that we're simulating.

Reply:
alright, changed that.